docs/source/getting_started/benchmarks/cifar10_benchmark.py

```diff
@@ -138,29 +139,31 @@
 path_to_train = '/datasets/cifar10/train/'
 path_to_test = '/datasets/cifar10/test/'
 
-# Use SimCLR augmentations, additionally, disable blur for cifar10
-collate_fn = lightly.data.SimCLRCollateFunction(
-    input_size=32,
-    gaussian_blur=0.,
+# Collate function init
+collate_fn = MultiViewCollate()
+
+# Use SimCLR augmentations
+simclr_transform = lightly.transforms.simclr_transform.SimCLRTransform(
```

Comment:
I'd just use `SimCLRTransform` here and make sure you don't import `lightly.transforms.simclr_transform.SimCLRTransform` but from `lightly.transforms import SimCLRTransform`. Because the way the new transforms are built it just creates very long imports which is not very nice.

Reply:
I followed how it was done before in the benchmarks. I'm fixing that.